src/pages/workspaces/workspace/Data.js

```diff
@@ -450,7 +450,7 @@ const WorkspaceData = _.flow(
   const getRunningImportJobs = async () => {
     try {
       const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true)
-      _.map(job => {
+      _.forEach(job => {
```

Comment:
The linter was complaining about this map having no return - I figured if there was no return expected here, this would essentially be running a forEach.

Let me know if I should revert!

Reply:
Thanks for this.

Reply:
The conversion makes sense. Because what's happening within the `forEach` is an effect, it may make sense to utilize `useEffect()`. We can revisit this later.